ES translations and menu updates to show ES

KimbaNessie opened the pull request:
Spanish translation of Treatment section, minor typos in Am I trans? Spanish version, inclusion of Spanish in the language menu, inclusion of numbered section index within the index, and minor corrections to dropdown section list in menu.

Twipped commented:
I was under the impression there were still major sections of the spanish translation missing (not being able to read it, I couldn't confirm). Did you review all the pages to confirm they're in sync?

KimbaNessie commented:
Hi,
I did not realise there were a couple more sections missing, namely, the
last few paragraphs of Causes and Androgenic Second Puberty. The section I
translated was not listed in the menu and that's how I found out about it,
so I wrongly assumed the other sections had been translated already.
I will translate them asap. If it's more efficient for you, reject my pull
request and I'll send an updated version when I have them.
Best!

On Tue, 14 Mar 2023 at 19:09, Jocelyn Badgley ***@***.***>
wrote:

> I was under the impression there were still major sections of the spanish
> translation missing (not being able to read it, I couldn't confirm). Did
> you review all the pages to confirm they're in sync?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/GenderDysphoria/GenderDysphoria.fyi/pull/128#issuecomment-1469087305>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADLYXRJBAXFNWIGRHR3AHZLW4EJE7ANCNFSM6AAAAAAV24SLQU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Fernando Gual Suárez
No. cuenta UNAM: 416003503

Twipped commented:
No need to reject, just go ahead and make the changes on this branch and let me know when you're done ❤️ 

There is actually already a finished translation, but it needed more work to get it integrated into the site code, and I haven't had the spoons to tackle it. https://drive.google.com/file/d/1mdX-1Mdl_UluCO4rzSuwtyj5jLHyMyIe/view?usp=sharing

KimbaNessie commented:
Ok, so I finished integrating the missing pages and translating the
sections I mentioned. Also, there was an entire section of the conclusion
missing. Added the correct links for siblings on each section and checked
the order. Also added the language tag for the menu in a few sections where
it was missing. I have committed these changes to this branch.

I should also mention: in the Spanish version there are duplicates of two
sections: (disforia-existencial, historico-disforia, and disforia-historica
(all three correspond to existential-dysphoria), and disforia-controlada
and disforia-gestionada (which correspond to managed-dysphoria)). I left
the page sequence as it is, using disforia-historica for existential
dysphoria and disforia-gestionada for managed dysphoria. The other versions
are redundant and mostly very similar, but I don't think I have the
authority to decide which translated version should stay 😅. In a future
edit, I could go over the entire thing and make the language uniform, and
the decision could be based on the resulting style.

Best!


On Wed, 15 Mar 2023 at 09:49, Jocelyn Badgley ***@***.***>
wrote:

> No need to reject, just go ahead and make the changes on this branch and
> let me know when you're done ❤️
>
> There is actually already a finished translation, but it needed more work
> to get it integrated into the site code, and I haven't had the spoons to
> tackle it.
> https://drive.google.com/file/d/1mdX-1Mdl_UluCO4rzSuwtyj5jLHyMyIe/view?usp=sharing
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/GenderDysphoria/GenderDysphoria.fyi/pull/128#issuecomment-1470289770>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADLYXRM4WNMNQHAA3P5XK33W4HQHTANCNFSM6AAAAAAV24SLQU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Fernando Gual Suárez
No. cuenta UNAM: 416003503

Twipped commented:
The existential dysphoria page used to be called historical dysphoria. The word never really sat right with me, and I changed it a few months after first publishing it. So "existencial" is the correct term to use.

That gives you a clue just how old the incomplete translation is 😄

KimbaNessie commented:
Ok, perfect. I have updated the siblings and the menu links to reflect
this. The disforia-gestionada version of managed-dysphoria appears to me as
a more natural translation, and it uses neutral language, so I left it as
is. So the sequence in the branch is now
disforia-de-presentacion->disforia-existencial->disforia-gestionada.

On Thu, 16 Mar 2023 at 08:17, Jocelyn Badgley ***@***.***>
wrote:

> The existential dysphoria page used to be called historical dysphoria. The
> word never really sat right with me, and I changed it a few months after
> first publishing it. So "existencial" is the correct term to use.
>
> That gives you a clue just how old the incomplete translation is 😄
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/GenderDysphoria/GenderDysphoria.fyi/pull/128#issuecomment-1472069024>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADLYXRLRDGRUVUEYOH4FI7TW4MOITANCNFSM6AAAAAAV24SLQU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Fernando Gual Suárez
No. cuenta UNAM: 416003503

KimbaNessie commented:
:D
Thanks! Hope to proofread the whole thing soon-ish.
